pages/SolCreateCheckYourAnswersPage.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
     I.seeInCurrentUrl('solicitorCreate/submit');
     I.see('Check your answers');
     I.see('Check the information below carefully. This will form a record of your application.');
-    I.waitForVisible(this.fields.submit, 10);
+    I.waitForVisible(this.fields.submit, 20);
```

Comment:
Should we have this in a config so we can change it in one place next time?

Reply:
possibly yes